chalk.nimble

```diff
@@ -8,10 +8,9 @@ bin           = @["chalk"]
 
 # Dependencies
 requires "nim >= 2.0.0"
-requires "https://github.com/crashappsec/con4m#8ab25930d193b9a11b3cd15531b252b25e28ff47"
-
-requires "https://github.com/viega/zippy == 0.10.7"
-requires "https://github.com/aruZeta/QRgen == 3.0.0"
+requires "https://github.com/crashappsec/con4m#f68d00c106a2b6cbd7134e0817c9b60852c4beee"
+requires "https://github.com/viega/zippy == 0.10.7" # MIT
+requires "https://github.com/aruZeta/QRgen == 3.0.0" # MIT
```

Comment:
Are the license comments really necessary? :) They probably will get rm'd in a future update.

Reply:
was an artifact from the context PR where I added them initially but dont think they hurt anything